_pages/install/site.adoc

```diff
@@ -0,0 +1,348 @@
+filenames across all output formats.
+
+For the list of available variables,
+see the <<_available_variables>> section.
+
+=== Default Behavior
+
+When an empty string is passed as the template (or when no template is
+specified),
+the system generates a normalized version of the document identifier.
+
+[example]
+====
+For a document with identifier "ISO/IEC FDIS 12345-3",
+the default output would be:
+
+[source]
+----
+iso-iec-fdis-12345-3
+----
+====
+
+=== Available Variables
+
+Templates can access document metadata through the `document` object:
+
+* `docidentifier` - Full document identifier
+* `language` - Document language code
+* `edition` - Edition number
+* `doctype` - Document type
+* `docnumber` - Document number
+* `partnumber` - Part number (if applicable)
+
+=== Basic Template Example
+
+[source,yaml]
+----
+metanorma:
+  template:
+    output_filename: "{{ document.docidentifier | downcase | replace: '/' , '-' }}"
+----
+
+=== Conditional Template Example
+
+[source,yaml]
+----
+metanorma:
+  template:
+    output_filename: |
+      {%- if document.doctype == 'international-standard' -%}
+        iso-
+      {%- else -%}
+        std-
+      {%- endif -%}
+      {{- document.docnumber -}}
+      {%- if document.partnumber %}-{{ document.partnumber }}{% endif %}
+----
+
+This demonstrates:
+
+* link:https://shopify.github.io/liquid/tags/control-flow/[Conditional logic^]
+  based on document type
+* Optional part number inclusion
+* Multi-line template formatting
+  * link:https://shopify.github.io/liquid/basics/whitespace/[White-space trimming^]
+    with `{%-`, `-%}`, and `{{-`, `-}}`
+
+=== Template Filters
+
+Common Liquid filters available:
+
+* link:https://shopify.github.io/liquid/filters/downcase/[`downcase`^] - Convert to lowercase
+* link:https://shopify.github.io/liquid/filters/replace/[`replace`^] - String replacement
+* link:https://shopify.github.io/liquid/filters/strip/[`strip`^] - Remove leading/trailing whitespace
+
+=== Error Handling
+
+* Invalid syntax raises `Liquid::SyntaxError`
+* Missing variables are replaced with empty strings
+* Example with missing variable:
++
+[source,yaml]
+----
+# Template: "{{ nonexistent }}_{{ document.language }}"
+# Result: "_en"
+----
+
+== Site Structure
+
+=== Generation Process
+
+. Scans source directory for Metanorma documents
+. Compiles each document with specified options
+. Creates site directory structure
+. Generates collection index (documents.xml)
+. Creates HTML index page
+. Copies assets and compiled documents
+
+=== Output Directory Structure
+
+----
+_site/              # Default output directory
+├── index.html      # Main site index
+├── documents.xml   # Collection index
+└── documents/      # Compiled documents
+    ├── doc1.html
+    ├── doc1.pdf
+    ├── doc1.rxl
+    ├── doc1.xml
+    ├── doc2.html
+    ├── doc2.pdf
+    ├── doc2.rxl
+    └── doc2.xml
+----
+
+== Best Practices
+
+* Organize documents in logical directory structures
```

Comment:
Please provide examples for all of these, the user wants to know how to achieve "best practice".